src/aks-preview/azext_aks_preview/addonconfiguration.py

```diff
@@ -6,10 +6,16 @@
 import json
 from knack.log import get_logger
 from knack.util import CLIError
-from azure.cli.core.azclierror import ArgumentUsageError, ClientRequestError
+from azure.cli.core.azclierror import AzCLIError, ArgumentUsageError, ClientRequestError
```

Comment:
Some new but not used imports like `AzCLIError`, `send_raw_request`, `HttpResponseError`.

Reply:
> Some new but not used imports like `AzCLIError`, `send_raw_request`, `HttpResponseError`.

Fixed as well.